test/BasketHandlerEdgeCases.test.ts

```
@@ -0,0 +1,43 @@
+  });
+
+  describe("Critical Edge Case Validations", function () {
+    it("1. Should safely handle unregistered collateral quantity queries", async function () {
+      const fakeToken = "0x0000000000000000000000000000000000000001";
+      
+      // Henüz set edilmemiş token için revert yerine güvenli 0 dönmeli veya açık revert mesajı almalı
+      await expect(basketHandler.quantity(fakeToken)).to.be.reverted;
+    });
+
+    it("2. Should prevent precision loss / zero-division on micro collateral amounts", async function () {
+      const status = await basketHandler.status();
```

ESLint

[error] Delete `;`

(prettier/prettier)